src/App/components/Preview.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import classnames from 'classnames';
 import styled from 'styled-components';
+import _ from 'lodash';
```

Comment:
Can you load just the module you need rather than everything in lodash to keep our footprint down? We are doing that in other places but there a few recent changes that just grab all of lodash like here.